docs/third-party-packages.md

```diff
@@ -97,6 +97,15 @@ A plugin for providing an endpoint that exposes [Prometheus](https://prometheus.
 
 A simple tool for integrating Starlette and WTForms. It is modeled on the excellent Flask-WTF library.
 
+### Starlette-Login
+
+<a href="https://github.com/jockerz/Starlette-Login" target="_blank">GitHub</a> |
+<a href="https://starlette-login.readthedocs.io/en/stable/" target="_blank">Documentation</a>
```

Comment:
Is the link this one or https://starlette-login.readthedocs.io/en/latest/ ?

Reply:
Yes